.github/workflows/ci-cd.yml

```diff
@@ -115,6 +115,63 @@ jobs:
         with:
           run: yarn electron test
 
+  performance:
+    name: Performance Tests
```

Comment:
@cdamus I believe we should move the job to a new workflow (and consequently check) instead of trying to fit it in the existing CI/CD workflow:

- we would not want to affect the overall build status if something fails in the job
- we would not want to increase the time for builds (potentially up to 30 min more if we hit the timeout)
- the workflow already re-does work from the build (yarn and building examples)
- clearer and easier to maintain when separated into it's own workflow

Reply:
I would of course make sure that it is 1 additional CI check, not add noise by creating 1 additional check per operating system or node version.

Reply:
Thanks, @vince-fugnitto I think the rebase detected that what once was the only workflow file was transmuted into the `ci-cd.yml` file. I'll factor this out into a new file for better isolation and maintenance.